plutus-core/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog for plutus-core
```

Comment:
Is it worth adding these if we don't have any content yet? I think we should maybe start producing them before we do a release with a CHANGELOG that's basically TODO

Reply:
Yeah, they don't serve much useful purpose. removed.